.github/workflows/base-release.yaml

```diff
@@ -141,7 +165,7 @@ jobs:
           if-no-files-found: error
 
   release:
-    name: ${{ inputs.distribution }} Release
+    name: Release
```

Comment:
Don't we want to keep the distribution in the name anymore? I think it would even be nice to indicate whether it was a nightly release. 🤔